### DISCUSSION TITLE
Blank LXC With GPU Passthrough Template

### DISCUSSION THREAD

**Discussion by MoldyTaint** [tteck/Proxmox#1490] [Ideas]
I am very technically stupid when it comes to Proxmox. I can't get passthrough to save my life, but when I use the Plex script you provided I can get the transcoding working with the GPU passthrough. Something I do in creating an LXC makes me mess it up, your script does it well for me. Having a template with a blank Ubuntu/Debian LXC with GPU passthrough would do me and i'm sure others...wonderful things.

**Comment by tteck** [comment]
If the Plex LXC meets your needs satisfactorily, you can simply create a new Plex LXC instance and uninstall Plex Media Server using the command `apt autoremove --purge plexmediaserver`, which will leave you with a clean Ubuntu 22.04 LXC with passthrough capabilities.

**Comment by MoldyTaint** [comment]
Thank you for a great and timely reply and explanation to remedy the situation. Is there a way to even modfy the script on my own to where Plex and it's dependencies do not install?

**Reply by tteck** [reply]
The dependencies are the standard ones installed on all LXCs, any others are needed for GPU passthrough.

**Reply by MoldyTaint** [reply]
Oh I apologize, I must have not had it explained right, I meant just for Plex. If there's a way I'm able to remove the Plex part out of the LXC Script. Your scripts are so helpful, I don't want to request something that I may be able to do. Your first explanation of just uninstalling Plex was helpful. :)
